ApnsPHP/Message/Tests/CustomMessageGetPayloadDictionaryTest.php

```diff
@@ -15,7 +15,7 @@
  *
  * @covers \ApnsPHP\Message\CustomMessage
  */
-class CustomMessageGetPayloadDictionaryTest extends CustomMessageTest
+class CustomMessageGetPayloadDictionaryTest extends CustomMessageTestBase
```

Comment:
I don't know where this is coming from. Couldn't find anything in the PHPUnit release notes that indicates this is necessary

Reply:
I can't find it either, the closest I can find is:
> The tests for a class Greeter go into a class GreeterTest.

https://docs.phpunit.de/en/10.5/writing-tests-for-phpunit.html#asserting-return-values

But the reality is that you'll have an output full of "Class CustomMessageTest did not contain any tests" if you don't rename them.

Reply:
https://github.com/sebastianbergmann/phpunit/issues/5132 doesn't provide any reasoning either, as usual...

However, the general consensus seems to be to rename these classes to `*TestCase` rather than `*TestBase`, which I like slightly better as well.